frontends/web/src/routes/account/summary/chart.tsx

```diff
@@ -53,6 +53,35 @@ class Chart extends Component<Props, State> {
     };
 
     public componentDidMount() {
+        this.createChart();
+    }
+
+    public componentWillUnmount() {
+        window.removeEventListener('resize', this.onResize);
+        if (this.chart) {
+            this.chart.timeScale().unsubscribeVisibleLogicalRangeChange(this.calculateChange);
+        }
+    }
+
+    public componentDidUpdate(prev) {
+        const { dataDaily, dataHourly } = this.props;
+        if (!this.chart && dataDaily && dataHourly) {
+            this.createChart();
+        }
+        if (
+            (this.lineSeries && prev.dataDaily && prev.dataHourly && dataDaily && dataHourly)
+            && (
+                prev.dataDaily.length !== dataDaily.length
+                || prev.dataHourly.length !== dataHourly.length
+                // || prev.dataDaily[prev.dataDaily.length - 1].value !== dataDaily[dataDaily.length - 1].value
```

Comment:
remove comments

Reply:
I wasn't sure, so we are not going to have newest data in the array?
but then it will mess up the relative-percentage-value that is calculated a bit...

Reply:
ok I'll just remove

Reply:
>  but then it will mess up the relative-percentage-value that is calculated a bit...

it doesn't, it uses the total as shown above the chart.